allensdk/brain_observatory/behavior/data_objects/cell_specimen_table.py

```diff
@@ -0,0 +1,236 @@
+import numpy as np
+import pandas as pd
+from pynwb import NWBFile, ProcessingModule
+from pynwb.ophys import OpticalChannel, ImageSegmentation
+
+import allensdk.brain_observatory.roi_masks as roi
+from allensdk.brain_observatory.behavior.data_objects import DataObject
+from allensdk.brain_observatory.behavior.data_objects.base \
```

Comment:
collapsing these interfaces into one file, or, adding a little import content to the `__init__.py` could simplify your import boilerplate here and elsewhere.

Reply:
It is a bit verbose, but I should mention that these are all automatically generated with the help of the IDE and I don't type each import statement out

Reply:
ok, I'm slightly in favor of collapsing ABCs into one or two files, to reduce import boilerplate.
But, what you have is very explicit, so, it is not too bad. Just verbose.